### ISSUE TITLE
Crash Saat Klik Back di 3D Secure Pin

### ISSUE THREAD

**Issue by huteri** [veritrans/veritrans-android#277]
Payment dengan creditcard 4811 1111 1111 1114, crash saat klik back di halaman masukin pin 3D Secure

```

                                                                                 Process: com.sweetescape.android.staging, PID: 25893
                                                                                 java.lang.RuntimeException: Unable to resume activity {com.sweetescape.android.staging/com.midtrans.sdk.uikit.activities.CreditDebitCardFlowActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String com.midtrans.sdk.corekit.models.TransactionResponse.getOrderId()' on a null object reference
                                                                                     at android.app.ActivityThread.performResumeActivity(ActivityThread.java:4221)
                                                                                     at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:4323)
                                                                                     at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1865)
                                                                                     at android.os.Handler.dispatchMessage(Handler.java:102)
                                                                                     at android.os.Looper.loop(Looper.java:148)
                                                                                     at android.app.ActivityThread.main(ActivityThread.java:7325)
                                                                                     at java.lang.reflect.Method.invoke(Native Method)
                                                                                     at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1230)
                                                                                     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1120)
                                                                                  Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String com.midtrans.sdk.corekit.models.TransactionResponse.getOrderId()' on a null object reference
                                                                                     at com.midtrans.sdk.uikit.fragments.PaymentTransactionStatusFragment.setupStatusInfo(Unknown Source)
                                                                                     at com.midtrans.sdk.uikit.fragments.PaymentTransactionStatusFragment.setUiForFailure(Unknown Source)
                                                                                     at com.midtrans.sdk.uikit.fragments.PaymentTransactionStatusFragment.bindDataToView(Unknown Source)
                                                                                     at com.midtrans.sdk.uikit.fragments.PaymentTransactionStatusFragment.onViewCreated(Unknown Source)
                                                                                     at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1314)
                                                                                     at android.support.v4.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManager.java:1528)
                                                                                     at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1595)
                                                                                     at android.support.v4.app.BackStackRecord.executeOps(BackStackRecord.java:757)
                                                                                     at android.support.v4.app.FragmentManagerImpl.executeOps(FragmentManager.java:2355)
                                                                                     at android.support.v4.app.FragmentManagerImpl.executeOpsTogether(FragmentManager.java:2146)
                                                                                     at android.support.v4.app.FragmentManagerImpl.optimizeAndExecuteOps(FragmentManager.java:2098)
                                                                                     at android.support.v4.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:2008)
                                                                                     at android.support.v4.app.FragmentController.execPendingActions(FragmentController.java:388)
                                                                                     at android.support.v4.app.FragmentActivity.onStart(FragmentActivity.java:607)
                                                                                     at android.support.v7.app.AppCompatActivity.onStart(AppCompatActivity.java:178)
                                                                                     at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1266)
                                                                                     at android.app.Activity.performStart(Activity.java:6943)
                                                                                     at android.app.Activity.performRestart(Activity.java:7002)
                                                                                     at android.app.Activity.performResume(Activity.java:7007)
                                                                                     at android.app.ActivityThread.performResumeActivity(ActivityThread.java:4210)
                                                                                     at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:4323) 
                                                                                     at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1865) 
                                                                                     at android.os.Handler.dispatchMessage(Handler.java:102) 
                                                                                     at android.os.Looper.loop(Looper.java:148) 
                                                                                     at android.app.ActivityThread.main(ActivityThread.java:7325) 
                                                                                     at java.lang.reflect.Method.invoke(Native Method) 
                                                                                     at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1230) 
                                                                                     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1120) 
```

**Comment by huteri** [comment]
Tolong kalo sempat di cek

**Comment by rakawestu** [comment]
Ok, saya cek dulu ya.

**Comment by rakawestu** [comment]
@huteri udah released di versi `1.4.2` ya.

https://github.com/veritrans/veritrans-android/releases/tag/v1.4.2

**Comment by huteri** [comment]
Oke terima kasih @rakawestu

-- 
Huteri Manza

On March 16, 2017 at 18:25:41, Raka Westu Mogandhi (notifications@github.com)
wrote:

@huteri <https://github.com/huteri> udah released di versi 1.4.2 ya.

https://github.com/veritrans/veritrans-android/releases/tag/v1.4.2

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/veritrans/veritrans-android/issues/277#issuecomment-287029481>,
or mute the thread
